e2e/features/record-agreed-hearing-requirements.feature

```diff
@@ -134,6 +137,9 @@ Feature: Record agreed hearing requirements
     And I should see the text `Do not enter the reason the appellant made the request.`
     And I should see the text `The respondent will be able to see this information and must not be informed of the appellant's personal circumstances.`
     And I should see the text `Adjustment request`
+
+    And I record `true` for Remote hearing
```

Comment:
Possibly more descriptive to have the text as `Remote hearing requirement will be reviewed`